src/datasources/targeted-messaging/targeted-messaging.datasource.spec.ts

```diff
@@ -115,6 +115,7 @@ describe('TargetedMessagingDataSource tests', () => {
         sourceFile: dto.sourceFile,
         sourceFileProcessedDate: null,
         sourceFileChecksum: null,
+        targetAll: false,
```

Comment:
Here we can access the value from the `DTO`. And making the DTO random would help test more scenarios.

Reply:
https://github.com/safe-global/safe-client-gateway/commit/1b72d7acb028af5275397cac1a6b18e63e85e435#diff-ae1f316c87534e60c2f465f11b0301577d8eaf3b6505da5f0bbc49fe31d3fe07L196